.github/workflows/ci-build.yaml

```diff
@@ -21,10 +21,6 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v2
-        with:
```

Comment:
@simster7 thoughts on this change?

Reply:
All good with me as long as it plays nice with `GIT_BRANCH             = $(shell git branch --show-current)`, which was the original reason for the code being this way